qdrant-landing/content/documentation/concepts/search.md

```diff
@@ -209,7 +209,33 @@ client.search(
 )
 ```
 
-Parameter `with_payload` might also be used to `include` or `exclude` specific fields only:
+The parameter `with_payload` may also be used to scope to or filter out a
+specific payload subset. You can specify an array of items to include. For
+example, to include `city`:
+
+```http
+POST /collections/{collection_name}/points/search
+
+{
+    "vector": [0.2, 0.1, 0.9, 0.7],
+    "with_payload": ["city"]
```

Comment:
Since you suggested that users can specify an array of items (line 213), I'd include an array `with_payload`, maybe something like:

```suggestion
    "with_payload": ["city", "village", "town"]
```